<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_teacher</class>
 <widget class="QMainWindow" name="MainWindow_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师用户</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_teacher">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>651</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_course_o">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>必修课程列表</string>
     </attribute>
     <widget class="QTableView" name="tableView_course_o">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>461</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>105</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_course_e">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>已选课程列表</string>
     </attribute>
     <widget class="QTableView" name="tableView_course_e">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>661</width>
        <height>441</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>105</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_course_s">
     <attribute name="title">
      <string>选课列表</string>
     </attribute>
     <widget class="QTableView" name="tableView_course_s">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>441</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>105</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;教师信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_login"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>课程</string>
    </property>
    <addaction name="action_course_o"/>
    <addaction name="action_course_e"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_password"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_login">
   <property name="text">
    <string>重新登录</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_course_o">
   <property name="text">
    <string>查看必修课程</string>
   </property>
  </action>
  <action name="action_course_e">
   <property name="text">
    <string>查看选修课程</string>
   </property>
  </action>
  <action name="action_password">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>查看选课列表</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
